io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/writer/internal/StreamGmlWriter.java

```diff
@@ -511,6 +511,9 @@ public boolean isPassthrough() {
 			InstanceCollectionPartitioner partitioner = getPartitioner(this, SimpleLog.NO_LOG);
 			return !partitioner.usesReferences();
 		}
+		else if (isPartitionByFeatureTypeConfigured()) {
```

Comment:
So `isThresholdConfigured()` and `isPartitionByFeatureTypeConfigured()` cannot be true at the same time?
Otherwise I would move the `isPartitionByFeatureTypeConfigured()` condition to be the first check.

Reply:
They can't be at the moment, but I'll move it anyways in case that changes in the future.